doc/source/user_guide/hpc/launch_mapdl_entrypoint.rst

```diff
@@ -0,0 +1,235 @@
+    cannot be used again.
+
+The values of each scheduler argument are wrapped with single quotes (`'`).
+This might cause parsing issues which can cause the job to fail after successful
+submission.
+
+It should be noticed also that PyMAPDL passes all the environment variables of the
+user to the new job and to the MAPDL instance.
+This is normally convenient, since there are many environmental variables which are
+needed to run the job or MAPDL.
+For instance the license server is normally stored in :envvar:`ANSYSLMD_LICENSE_FILE` environment variable.
+If you prefer not to pass these environment variables to the job, use the SLURM argument
+``--export`` to specify the desired environment variables.
+For more information, see `SLURM documentation <slurm_docs_>`_.
+
+
+Working with the instance
+-------------------------
+
+Once the :class:`Mapdl <ansys.mapdl.core.mapdl.MapdlBase>` object has been created,
+there are no differences with a normal :class:`Mapdl <ansys.mapdl.core.mapdl.MapdlBase>`
+instance.
+You can retrieve the IP of the MAPDL instance, as well as its hostname.
+
+.. code:: pycon
+
+    >>> mapdl.ip
+    '123.45.67.89'
+    >>> mapdl.hostname
+    'node0'
+
+You can also retrieve the job ID:
+
+.. code:: pycon
+
+    >>> mapdl.jobid
+    10001
+
+If you want to check whether the instance has been launched using an scheduler,
+you can use :attr:`mapdl_on_hpc <ansys.mapdl.core.mapdl_grpc.MapdlGrpc.mapdl_on_hpc>`
+
+.. code:: pycon
+
+    >>> mapdl.mapdl_on_hpc
+    True
+
+
+Sharing files
+^^^^^^^^^^^^^
+
+Most of the HPC cluster share the login node filesystem with the compute nodes,
+meaning you do not need to do extra work to upload or download files to the MAPDL
+instance, you just need to copy them to the location where MAPDL is running.
+This location can be obtained with
+:attr:`directory <ansys.mapdl.core.mapdl_grpc.MapdlGrpc.directory>`.
+
+If no location is specified in :func:`launch_mapdl() <ansys.mapdl.core.launcher.launch_mapdl>`,
+then a temporal location is selected.
+It is a good idea to set the ``run_location`` argument to a directory that is accessible
+from all the compute nodes.
+Normally anything under ``/home/user`` is available to all compute nodes.
+In case you are unsure where you should launch MAPDL, contact your cluster administrator.
+
+Additionally, you can use methods like :meth:`upload <ansys.mapdl.core.mapdl_grpc.MapdlGrpc.upload>`
+or :meth:`download <ansys.mapdl.core.mapdl_grpc.MapdlGrpc.download>` to
+upload or download files to/from the MAPDL instance respectively.
+You do not need other connection like ``ssh`` or similar.
+However, for large files, you might want to consider other alternatives.
+
+
+Exiting MAPDL
+-------------
+
+Exiting MAPDL, either intentionally or unintentionally, stop the job.
+This behaviour is because MAPDL is the main process at the job, hence when finished
+the scheduler considers the job done.
```

Comment:
```suggestion
Exiting MAPDL, either intentionally or unintentionally, stops the job.
This behavior occurs because MAPDL is the main process at the job. Thus, when finished,
the scheduler considers the job done.
```